server/tests-py/test_schema_stitching.py

```diff
@@ -453,12 +455,90 @@ def test_remote_query_timeout(self, hge_ctx):
         # wait for graphql server to finish else teardown throws
         time.sleep(6)
 
-#    def test_remote_query_variables(self, hge_ctx):
-#        pass
-#    def test_add_schema_url_from_env(self, hge_ctx):
-#        pass
-#    def test_add_schema_header_from_env(self, hge_ctx):
-#        pass
+@pytest.mark.skipif(not pytest.config.getoption("--enable-remote-schema-permissions"),
+                    reason="flag --enable-remote-schema-permissions is not set. Cannot run tests for remote schema permissions")
+class TestCreateRemoteSchemaPermissions():
+    rs_dir = 'queries/remote_schemas/permissions/'
+    teardown = {"type": "remove_remote_schema", "args": {"name": "simple"}}
+
+    @pytest.fixture(autouse=True)
+    def transact(self, hge_ctx):
+        q = mk_add_remote_q('simple', 'http://localhost:5000/user-graphql')
+        st_code, resp = hge_ctx.v1q(q)
+        assert st_code == 200, resp
+        yield
+        st_code, resp = hge_ctx.v1q(self.teardown)
+        assert st_code == 200, resp
+
+    def test_no_roles(self, hge_ctx):
+        check_query_f(hge_ctx, self.rs_dir + 'basic_fail.yaml')
+
+    def test_basic(self, hge_ctx):
+        st_code, resp = hge_ctx.v1q_f(self.rs_dir + 'setup.yaml')
+        assert st_code == 200, resp
+        # make good query
+        check_query_f(hge_ctx, self.rs_dir + 'basic_success.yaml')
+        # make bad query
+        check_query_f(hge_ctx, self.rs_dir + 'basic_fail_2.yaml')
+        st_code, resp = hge_ctx.v1q_f(self.rs_dir + 'teardown.yaml')
+        assert st_code == 200, resp
+        check_query_f(hge_ctx, self.rs_dir + 'basic_fail.yaml')
```

Comment:
This test may be ignored, as it has already been done above.

Reply:
This is to check whether the query returns the same thing after removing the perms.